seedlet-blocks/sass/blocks/_latest-posts.scss

```
@@ -0,0 +1,125 @@
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
+
+.wp-block-latest-posts {
+	padding-left: 0;
+
+	// Vertical margins logic
+	&:not(.is-grid) > li {
+		margin-top: var(--wp--custom--margin--vertical);
+		margin-bottom: var(--wp--custom--margin--vertical);
+
+		&:first-child {
+			margin-top: 0;
+		}
+
+		&:last-child {
+			margin-bottom: 0;
+		}
+	}
+
+	&.is-grid {
+
+		& > li {
+			margin-bottom: var(--wp--custom--margin--vertical);
+
+			&:last-child {
+				margin-bottom: 0;
+			}
+		}
+
+		// Remove bottom margins in grid columns
+		&.columns-2 > li:nth-last-child(-n + 2):nth-child(2n + 1),
+		&.columns-2 > li:nth-last-child(-n + 2):nth-child(2n + 1) ~ li,
+		&.columns-3 > li:nth-last-child(-n + 3):nth-child(3n + 1),
+		&.columns-3 > li:nth-last-child(-n + 3):nth-child(3n + 1) ~ li,
+		&.columns-4 > li:nth-last-child(-n + 4):nth-child(4n + 1),
+		&.columns-4 > li:nth-last-child(-n + 4):nth-child(4n + 1) ~ li,
+		&.columns-5 > li:nth-last-child(-n + 5):nth-child(5n + 1),
+		&.columns-5 > li:nth-last-child(-n + 5):nth-child(5n + 1) ~ li,
+		&.columns-6 > li:nth-last-child(-n + 6):nth-child(6n + 1),
+		&.columns-6 > li:nth-last-child(-n + 6):nth-child(6n + 1) ~ li {
+			margin-bottom: 0;
+		}
+	}
+
+	& > li > * {
+		margin-top: calc(0.5 * var(--wp--custom--margin--vertical));
+		margin-bottom: calc(0.5 * var(--wp--custom--margin--vertical));
+
+		&:first-child {
+			margin-top: 0;
+		}
+
+		&:last-child {
+			margin-bottom: 0;
+		}
+	}
+
+	// Post title
+	& > li > a {
+		font-family: var(--wp--preset--font-family--headings);
+		line-height: 1.3;
+		font-size: 28px;
```

Thoughts about using a preset here instead? 

```suggestion
		font-size: var(--wp--preset--font-size--huge);
```

---

Yea, makes sense.